compiler/rustc_codegen_llvm/src/builder.rs

```diff
@@ -988,6 +988,13 @@ impl<'a, 'll, 'tcx> BuilderMethods<'a, 'tcx> for Builder<'a, 'll, 'tcx> {
         (self.extract_value(landing_pad, 0), self.extract_value(landing_pad, 1))
     }
 
+    fn filter_landing_pad(&mut self, pers_fn: &'ll Value) -> (&'ll Value, &'ll Value) {
+        let ty = self.type_struct(&[self.type_i8p(), self.type_i32()], false);
+        let landing_pad = self.landing_pad(ty, pers_fn, 1 /* FIXME should this be 0? */);
```

Comment:
Regarding the FIXME here: the number is a hint with the number of clauses to reserve space for (like `Vec::with_capacity`). 1 is the correct value to use here since we are only adding 1 clause.

Reply:
So the value for cleanup above should be 0?